### PR TITLE
Lint CSS files on precommit 

### PR DESCRIPTION
**Pull request by AlexeyKorkoza**
I described the rules in .csslintrc file. Then I run checking css code using csslint-cli. It check css code on default rules.
I want to check in the following rules which described in .csslintrc file. 
What is my mistake ?

### CODE REVIEW

**Comment by Vanish404** [comment]
@AlexeyKorkoza наверное, неправильно запускаешь. Судя по скрипту, ты  запускаешь `csslint .css`, а надо  `csslint-cli [options]* [file|dir [file|dir]]*` . Т.е. будет `csslint-cli .` либо  `csslint-cli ./css`  
Что насчет конфигурационного файла прочти [здесь](https://github.com/CSSLint/csslint/wiki/Command-line-interface#configuration-files). Еще вариант есть использовать ключ `--config=file_path`

**Comment by AlexeyKorkoza** [comment]
Спасибо за подсказу по запуску скрипта. Пишу в терминале WebStorm : csslint-cli ./css и скрипт проверяет css код по заданным правилам, которые прописаны в файле .csslintrc .

File: package.json [hunk 1]
```diff
@@ -22,7 +23,9 @@
   "homepage": "https://github.com/pub-t/plain-destination#readme",
   "devDependencies": {
     "concurrently": "^2.1.0",
-    "eslint": "^2.12.0",
+    "csslint": "^0.1.0",
+    "csslint-cli": "0.0.3",
```
Comment (irnc) [inline package.json]: Here you install alternative CLI, while at https://github.com/pub-t/plain-destination/pull/30#discussion-diff-68366948 you said that you intend to use the standard one. Please resolve that one.

File: package.json [hunk 2]
```diff
@@ -22,7 +23,9 @@
...
+    "eslint": "^2.13.1",
```
Comment (irnc) [inline package.json]: No need to update `eslint` in this branch.

File: package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
   "description": "Client part of the application. In this app you can find way to get to any point in the shortest time",
   "main": "index.js",
   "scripts": {
+    "csslint": "csslint .css",
```
Comment (irnc) [inline package.json]: I guess this invocation would result in no files linted, the same issues discussed at https://github.com/pub-t/plain-destination/pull/30#discussion-diff-68367081

File: .csslintrc [hunk 4]
```diff
@@ -0,0 +1,37 @@
+ {
+ "important": false,
```
Comment (irnc) [inline .csslintrc]: Please fix indentation.

File: .csslintrc [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+   "qualified-headings": false,
+   "selector-max": false,
+   "shorthand": true,
+   "text-indent": false,
+   "unique-headings": false,
+   "universal-selector": false,
+   "unqualified-attributes": false,
+   "vendor-prefix": false,
+   "zero-units": true
+}
```
Comment (irnc) [inline .csslintrc]: Please add newline here.

File: package.json (hunk 1 above)
Comment (AlexeyKorkoza) [inline package.json]: I decided what need that it is necessary use `csslint-cli` , because I installed `csscli-cli` .

File: package.json (hunk 3 above)
Comment (AlexeyKorkoza) [inline package.json]: How word a _**linted**_ translate on the Russian language?

Comment (irnc) [inline package.json]: Please check English meaning first, https://en.wiktionary.org/wiki/lint. Original meaning in Russian would be _корпия_, usually we don't translate it directly, but instead use it as neologism.

**Comment by KrivichaninDenis** [comment]
Why you merge this branch if csslint [script](https://github.com/pub-t/plain-destination/pull/37/commits/ac0883bf09503223855211bbcb19294c86dcf495#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L7) was deleted?

**Comment by AlexeyKorkoza** [comment]
I'm sorry. Now I merged only .csslintrc file with described rules in it.

**Pull request merged**